Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -432,14 +442,14 @@ class BocList
     }
   
     // Catch styles applied to pseudo-selectors starting at the last element in the DOM collection
-    tempList.insertAfter($(bocList));
+    bocList.parentNode!.insertBefore(tempList, bocList.nextSibling)
```

Comment:
What happens if boclist is the last element in the parent?
would https://www.w3schools.com/jsref/met_node_insertadjacentelement.asp be a better option? (then also for line 429)

@a-ctor

Reply:
It inserts at the end if `null` is supplied as reference node. So if no parent is found it behaves like `appendChild`.

Reply:
Ah, the docs about "Node" don't refer to "node.js".